api/v2/views/base.py

```diff
@@ -24,8 +24,9 @@ def unresolved_requests_only(fn):
     @wraps(fn)
     def wrapper(self, request, *args, **kwargs):
         instance = self.get_object()
+        user_can_act = request.user.is_staff or request.user.is_superuser or CloudAdministrator.objects.filter(user=request.user.id)
```

Comment:
@lenards you can't use .get because a "lookup miss" will generate a ModelDoesNotExist error.

So you are either dealing with `.filter()` logic 'in an if statement' or `.get()` logic inside 'try/except' statement..